db/migrate/20231028090921_create_comments.rb

```diff
@@ -0,0 +1,10 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.references :author, null: false, foreign_key: { to_table: 'users' }
```

Comment:
- [ ] In keeping with the guidelines given in the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd_v1_1.png), the foreign key should be `user_id` and not `author_id` as you have specified in your migration file above ☝️ 
### Only `Post` should have `author_id` 👍

Reply:
Hello @codepantha thank you for the review, let me correct this

Reply:
well resolved! ✔️